com.unity.uiwidgets/Runtime/engine2/UIWidgetsPanel.cs

```diff
@@ -172,33 +149,27 @@ public struct TextFont {
             base.OnEnable();
             D.assert(_wrapper == null);
             _wrapper = new UIWidgetsPanelWrapper();
-            var settings = new Dictionary<string, object>();
+            
             if (fonts != null && fonts.Length > 0) {
-                for (int i = 0; i < fonts.Length; i++) {
-                    if(!_internalTextFonts.ContainsKey(fonts[i].family))
-                        _internalTextFonts.Add(fonts[i].family,fonts[i]);
+                foreach (var font in fonts) {
```

Comment:
the logics is still not ok, what we need here is:

if _internalTextFonts contains three font families A, B, C(1), and fonts contains C(2). then we should pass the font families A, B, C(2) into _wrapper.Initiate